### PR TITLE
ISSUP-3265 End user is unable to add a comment on a ticket after initial creation

### PR DESCRIPTION
**Pull request by rdai10**
### Description
https://issues.liferay.com/browse/ISSUP-3265

### CODE REVIEW

File: theme/src/js/vendor/zendesk-default-script.js [hunk 1]
```diff
@@ -2,147 +2,135 @@
...
 
-		$('#quick-search').on(
-			'keypress',
-			function(event) {
-				if (event.which === 13) {
-					search();
-				}
-			}
-		);
+		setExpanded(userNav);
```
Comment (jonmak08) [inline theme/src/js/vendor/zendesk-default-script.js:107]: Should we check if `userNav` returns a DOM node before using in JS?

**Review by jonmak08 — COMMENTED** [review]
Hey @rdai10,

Just a minor update.

Please let me know if you have any questions. Thanks!

File: theme/src/js/vendor/zendesk-default-script.js (hunk 1 above)
Comment (rdai10) [reply theme/src/js/vendor/zendesk-default-script.js:107]: Sounds good

Comment (rdai10) [reply theme/src/js/vendor/zendesk-default-script.js:107]: Added at fd61930!

**Pull request merged**